config/software/libxslt.rb

```diff
@@ -43,12 +39,14 @@
 
   env = with_standard_compiler_flags(with_embedded_path({}, msys: true), bfd_flags: true)
 
-  patch source: "libxslt-cve-2015-7995.patch", env: env
   patch source: "libxslt-solaris-configure.patch", env: env if solaris?
-  patch source: "libxslt-mingw32.patch", env: env if windows?
 
+  # the libxslt configure script iterates directories specified in
+  # --with-libxml-prefix looking for the libxml2 config script. That
+  # iteration treats colons as a delimiter so we are using a cygwin
+  # style path to accomodate
   configure_commands = [
-    "--with-libxml-prefix=#{install_dir}/embedded",
+    "--with-libxml-prefix=#{install_dir.sub('C:', '/C')}/embedded",
```

Comment:
well i think this is peculiar to this one setting and think its generally something we dont want to do. i dont think its msys2 releated